scss/bootstrap/overrides/_variables.scss

```diff
@@ -23,10 +23,10 @@ $crm-page-topbar-bg-light:     #393E46;
 $crm-page-topbar-color:        #fff !default;
 $crm-page-topbar-item-padding: 15px !default;
 $crm-accordion-header-color:   #f3f6f7;
-$crm-dasboard-header:          #5D677B;
+$crm-dasboard-header:          #4D4D69;
```

Comment:
1) the variable has a typo
2) where is this used?
3) why are we defining a custom color for this instead of using a color from the palette?